tests/09-mdbook-mode/input/src/ch01-00-getting-started.md

```diff
@@ -0,0 +1,4 @@
+# Getting Started
```

Comment:
I think lets leave only a few representative files: each file must check some aspect we want to test, else the JSON is big for no reason.

Reply:
Sure